packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -405,17 +391,12 @@ class PerformanceMonitoring {
     return spans
   }
 
-  checkBrowserResponsiveness(transaction, interval, buffer) {
-    var counter = transaction.browserResponsivenessCounter
-    if (typeof interval === 'undefined' || typeof counter === 'undefined') {
-      return true
-    }
-
-    var duration = transaction.duration()
-    var expectedCount = Math.floor(duration / interval)
-    var wasBrowserResponsive = counter + buffer >= expectedCount
+  checkBrowserResponsiveness(transaction) {
+    const counter = transaction.browserResponsivenessCounter
+    const duration = transaction.duration()
+    const expectedCount = Math.floor(duration / BROWSER_RESPONSIVENESS_INTERVAL)
 
-    return wasBrowserResponsive
+    return counter + BROWSER_RESPONSIVENESS_BUFFER >= expectedCount
```

Comment:
The same here for function arguments.
Image if we want to test this function with different intervals or different buffers!